Testscripts/Linux/check_IB_SRIOV.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+# This script checks HPC VM prerequitions like waagent config, IB over ND HPC VM and IB over SR-IOV HPC
+# VM have different conditions to prove their setting right. Use the common library function call, and
+# get the HPC VM type. It also checks if there is anything missing in HPC VM requirements.
+########################################################################################################
+# Source utils.sh
+. utils.sh || {
+	echo "Error: unable to source utils.sh!"
+	echo "TestAborted" > state.txt
+	exit 0
+}
+# Source constants file and initialize most common variables
+UtilsInit
+
+# Constants/Globals
+# Get distro information
+GetDistro
+
+function Main() {
+
+	is_hpc_vm
+	_type=$?
+	if [ $_type == "3" ]; then
+		LogMsg "Could not determine VM type is either HPC or non-HPC."
+		SetTestStateFailed
+		exit 0
+	fi
+
+	if [ $_type == "2" ]; then
+		LogMsg "This VM is non-HPC VM. No further testing."
+		SetTestStateAborted
```

Comment:
Why aborted ? This should be skipped.

Reply:
Changed in 31395b019805889fb95966e52e9e8a51c54915e7